product_docs/docs/biganimal/release/using_cluster/02_connecting_your_cluster/02_connecting_from_aws/index.mdx

```diff
@@ -4,9 +4,52 @@ redirects:
 - /biganimal/release/using_cluster/02_connecting_your_cluster/02_connecting_from_aws/01_vpc_endpoint/
 ---
 
+
+
+AWS VPC endpoint (AWS Private Link) service is a network interface that securely connects a private IP address from your AWS VPC to an external service. You grant access only to a single cluster instead of the entire BigAnimal resource VPC, thus ensuring maximum network isolation. Other advantages include:
+
+-   You need to configure the PrivateLink only once. Then you can use multiple VPC endpoints to connect applications from different VPCs.
+-   There's no risk of IP address conflicts.
+
+There's an associated cost of resources, however.
+
+For more information, see [VPC endpoint services (AWS PrivateLink)](https://docs.aws.amazon.com/vpc/latest/privatelink/endpoint-service.html).
+
+The way you create a private endpoint differs when you are using your own AWS account as opposed to using BigAnimal's cloud account.
+
+## Using BigAnimal's cloud account
+
+When using BigAnimal's cloud account, you provide BigAnimal with your AWS account ID when creating a cluster. BigAnimal, in turn, provides you with a AWS service name, which you can use to connect to your cluster privately.
+
+1. When creating your cluster, on the **Cluster Settings** tab, in the **Network Connectivity** section:
+   1. Select **Private**.
+
+   1. Enter your application's AWS account ID.
+
+1. After the cluster is created, go to its cluster details to see the corresponding endpoint service name. You need the service name while creating a VPC endpoint.
+
+1. Create a VPC endpoint in the client's VPC. The steps for creating a VPC endpoint in the client's VPC are the same  whether you are using BigAnimal's cloud or your own. The steps are available [here](#step-2-create-a-vpc-endpoint-in-the-clients-vpc).
+
+1. In your application's AWS account, select **VPC** and then select **Endpoints**. Select the endpoint you created previously and use the service name provided in the details section in BigAnimal to access your cluster.
+
+   ```shell     
```

Comment:
what are we attempting to show the customer here?

Reply:
I suppose we wanted to show an example on how to connect, but I just realized that the example is wrong because it uses the AWS provided FQDN instead of the FQDN that BigAnimal provides (and we should use the latter because of the TLS certificate being issued for our domain, not the AWS one). I agree that we should either remove or rephrase this step...

Reply:
So is the consensus to take out the following code block? 
```shell     
   # Replace the port 5432 below if that has been changed in the Big Animal DB Configuration
   psql -h vpce-XXXXXXXXXXXXXXXXXXXX.eu-west-1.vpce.amazonaws.com -U edb_admin -p 5432     
    __OUTPUT__
    Password for user edb_admin: 
    ```
    ``` 
    psql (15.2)
    SSL connection (protocol: TLSv1.3, cipher: TLS_AES_256_GCM_SHA384, compression: off)
    Type "help" for help.

    edb_admin=>
    ```